src/librustc_mir/error_codes.rs

```diff
@@ -1993,6 +1993,68 @@ fn get_owned_iterator() -> IntoIter<i32> {
 ```
 "##,
 
+E0524: r##"
+A variable which requires unique access is being used in more than one closure
+at the same time.
+
+Erroneous code example:
+
+```compile_fail,E0524
+fn set(x: &mut isize) {
+    *x += 4;
+}
+
+fn dragoooon(x: &mut isize) {
+    let mut c1 = || set(x);
+    let mut c2 = || set(x); // error!
+
+    c2();
+    c1();
+}
+```
+
+To solve this issue, multiple solutions are available. First, is it required
+for this variable to be used in more than one closure at a time? If it is the
+case, use reference countered types such as `Rc` (or `Arc` if it runs
```

Comment:
```suggestion
case, use reference counted types such as `Rc` (or `Arc` if it runs
```
Oops, looks I just exchanged one typo for another. It's right now at least.